### PR TITLE
Add twitter icon to footer

### LINKED ISSUE #1375: Missing link to Twitter
## Where to find the issue

https://www.coronawarn.app/

## Describe the issue

https://www.coronawarn.app/ does not link to the Twitter channel https://twitter.com/coronawarnapp.

image

Related to https://github.com/corona-warn-app/cwa-wishlist/issues/364 "Use your Twitter-Account".

## Suggested change

Add link to https://twitter.com/coronawarnapp on footer of https://www.coronawarn.app/.

https://publish.twitter.com/ offers some automation possibilities.

I would like to leave this change to the Open Source team, so I wasn't planning to submit a PR on it.

---
Internal Tracking ID: [EXPOSUREAPP-8089](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-8089)

### PR DESCRIPTION
**Pull request by MikeMcC399**
This PR resolves #1375 "Missing link to Twitter". The link is added under "Learn more".

image

Twitter specifies an minimum of 32px for the size of the logo, which means it is larger than the GitHub logo in the footer.
https://about.twitter.com/content/dam/about-twitter/en/brand-toolkit/downloads/twitter-external-brand-guidelines-01272021.pdf.

The logo is from https://about.twitter.com/content/dam/about-twitter/en/brand-toolkit/downloads/twitter-logo-01282021.zip and I shrunk a .png file to do this.

### CODE REVIEW

File: src/data/global.json [hunk 1]
```diff
@@ -277,6 +285,14 @@
                         {
                             "title": "FAQ",
                             "url": "/de/faq/"
+                        },
+                        {
+                            "id": "twitter",
+                            "icon": "twitter",
+                            "icon-url": "/assets/img/icons/twitter_white.png",
+                            "title": "Das Corona-Warn-App-Project auf Twitter",
```
Comment (Ein-Tim) [inline src/data/global.json:293]: Was it intended to use "Project" in the German version too?

```suggestion
                            "title": "Das Corona-Warn-App-Projekt auf Twitter",
```

**Review by Ein-Tim — COMMENTED** [review]
Thank you so much! This is really a great addition! One small comment 😉

**Comment by MikeMcC399** [comment]
@Ein-Tim 
Thanks for noticing the typo. I didn't see anywhere where this text is actually used (maybe you know if it shows somewhere). I have corrected it anyway.

Let's see what @dsarkar thinks about if it looks OK.

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@MikeMcC399 @Ein-Tim Thanks!  Merged. LGTM. Maybe we increase the GitHub logo?

**Comment by Ein-Tim** [comment]
@dsarkar #1453 (-;

**Pull request merged**